exercise4.js

```diff
@@ -0,0 +1,3 @@
+function findEvenNumbers(numbers) {
+  return numbers.filter((number) => number % 2 === 0);
```

Comment:
No need for parenthesis when you have only one parameter is an arrow function but it is still fine to have it there if you prefer